devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -319,7 +369,23 @@ public void ClearDirectAligned(int x, int y, int width, int height)
         /// <param name="inverted">Indicates if color to be used turn the pixel on, or leave off.</param>
         public void DrawPixel(int x, int y, bool inverted = true)
         {
-            if ((x >= Width) || (y >= Height))
+            switch (Rotation)
+            {
+                case RotationOptions.Rotate90Degrees:
```

Comment:
you're missing the no rotation case. In case I'm rotating few times, if I then rotate again to no rotation, this may not work properly?

Reply:
The enum is a property in reference to the default orientation of the display and not a method that rotates it each time it's called.  You set the property once and it's done.  In fact, it may be useful to make it part of the constructor.  

If there is no rotation (default value of the Rotation property) then no byte switching is necessary.  Nothing to be done.